packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/comfortExchange/pack-xdsl-access-comfort-exchange.constant.js

```diff
@@ -0,0 +1,18 @@
+export const COMFORT_EXCHANGE_TYPE_ERROR = {
```

Comment:
Why do we need this ?

Reply:
To map with error code returned by the API

Reply:
Yes, I understood that, but why do you need an array to do that?

Maybe we can concat the error code with the translation directly, and if the translation doesn’t exist for an error, we can use a generic translation.

As example, you can compare the translated key and the result, if result === key, the translation doesn’t exist and you can use a fallback message.

If one day, we have a new kind of errors, we will avoid patching this array and the translation file. We will patch only one file.

What do you think about it?

Reply:
as you want

Reply:
sorry, I just use errBase and errContactShipping. Others error numbers are mapping with translations.